frontend/src/metabase/dashboard/context/context.tsx

```diff
@@ -339,6 +354,11 @@ const DashboardContextProviderInner = ({
     return dashboard;
   }, [dashboard, isDashcardVisible]);
 
+  const dashboardActions =
+    typeof initDashboardActions === "function"
+      ? initDashboardActions({ isEditing, downloadsEnabled })
```

Comment:
I mean this signature is oddly specific, I know it's cater to the previous version of the `EditableDashboard`, I don't have any better alternative, but I wanna raise my concern on the API.